### ISSUE TITLE
v1.92 (Windows): Dual mining etc+vtc with SSL not working (no port in API for 2nd algorithm)

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#369]
`TeamBlackMiner-v1.92\TBMiner.exe --algo eth+vtc --verthash-data ..\..\Cache\VertHash.dat --hostname daggerhashimoto.auto.nicehash.com --wallet ###############.########## --ssl-port 443 --server-passwd x --vtc-hostname verthash.mining-dutch.nl --vtc-wallet ##################.######### --vtc-port 9991 --ssl --ssl-verify-none --api --api-version 1.4 --api-port 4001 --cl-devices [0]
`
I get 0 hat rate for 2nd algorithm:

```
00:00:00 [2023-02-24 17:43:13.158]
00:00:00 [2023-02-24 17:43:13.290] OpenCL driver version: 527.56
00:00:00 [2023-02-24 17:43:13.324] Cuda driver version: 12.0
00:00:00 [2023-02-24 17:43:13.324] Cuda runtime API version: 12.0
00:00:00 [2023-02-24 17:43:13.324]
00:00:00 [2023-02-24 17:43:13.324]
00:00:00 [2023-02-24 17:43:13.324]
00:00:00 [2023-02-24 17:43:13.324] Welcome to Team Black Miner 1.92
00:00:00 [2023-02-24 17:43:13.324]
00:00:00 [2023-02-24 17:43:13.324]
00:00:00 [2023-02-24 17:43:13.324] GPU0 gfx1010:xnack- 8GB added to miner as OpenCL device
00:00:00 [2023-02-24 17:43:13.325] Trying to read verthash data file from disk
00:00:00 [2023-02-24 17:43:13.326] Api server started at 127.0.0.1:4001
00:00:00 [2023-02-24 17:43:13.995] Verthash data file has been loaded succesfully
00:00:00 [2023-02-24 17:43:19.355] Verthash data file has been verified succesfully
00:00:00 [2023-02-24 17:43:19.355] GPU0 Started CL work thread
00:00:00 [2023-02-24 17:43:19.355] Trying stratum+ssl://verthash.mining-dutch.nl:9991 (verthash)
00:00:00 [2023-02-24 17:43:19.355] GPU0 Intensity set to 256
00:00:00 [2023-02-24 17:43:19.355] Trying stratum+ssl://daggerhashimoto.auto.nicehash.com:443 (ethash)
00:00:00 [2023-02-24 17:43:19.411] daggerhashimoto.auto.nicehash.com: Pool responded successfully to subscribe
```

Pls. note that there is no port number in the API data:
```
{
  "pool": {
    "1": {
      "algo": "verthash",
      "block": 0,
      "diff": 0,
      "epoch": 0,
      "job": "",
      "ping": 0,
      "port": 9991,
      "total_accepted": 0,
      "total_hashrate": 0,
      "total_rejected": 0,
      "total_stale": 0,
      "url": "verthash.mining-dutch.nl",
      "worker": ""
    },
    "2": {
      "algo": "ethash",
      "block": 0,
      "diff": 0.478452,
      "epoch": 553,
      "job": "000000004fa4ce83",
      "ping": 24,
      "port": 0,
      "total_accepted": 0,
      "total_hashrate": 13089645,
      "total_rejected": 0,
      "total_stale": 0,
      "url": "daggerhashimoto.auto.nicehash.com",
      "worker": ""
    }
  }
}
```

What am I doing wrong?

Non-SSL mining at the same pools works fine.

**Comment by UselessGuru** [comment]
Any news?

**Comment by UselessGuru** [comment]
Any news?

**Comment by UselessGuru** [comment]
Solved in 1.94
